ui/apps/platform/src/Components/FilteredWorkflowViewSelector/FilteredWorkflowViewSelector.tsx

```diff
@@ -52,6 +52,7 @@ function FilteredWorkflowViewSelector({
                 </MenuToggle>
             )}
             shouldFocusToggleOnSelect
+            aria-label="Filtered workflow select options"
```

Comment:
Can you double-check that this is OK in the Element pane and/or aXe dev tools? I seem to remember a recent issue where the aria-label need to go on the `SelectList` component instead of the top level `<Select>`.